includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1064,17 +1064,12 @@ protected function get_wizard_in_cart_payment_gateways() {
 				'enabled' => $can_stripe,
 				'featured' => true,
 			),
-			'braintree_paypal' => array(
```

Comment:
There is a `$paypal_bt_description` further up on line 1031 that should be removed also.